docs/build/smart-contracts/getting-started/storing-data.mdx

```diff
@@ -27,12 +27,12 @@ This tutorial assumes that you've already completed the previous steps in Gettin
 
 ## Adding the increment contract
 
-The `stellar contract init` command allows us to initialize a new project with any of the example contracts from the [soroban-examples](https://github.com/stellar/soroban-examples) repo, using the `--with-example` (or `-w`) flag.
+The `stellar contract init` command allows us to initialize a new project with any of the example contracts from the [soroban-examples](https://github.com/stellar/soroban-examples) repo, using the `--name` flag, to specify a new contract's name.
 
-It will not overwrite existing files, so we can also use this command to add a new contract to an existing project. Run the command again with a `--with-example` flag to add an `increment` contract to our project. From inside our `soroban-hello-world` directory, run:
+It will not overwrite existing files unless we explicily pass in `--ovewrite` flag. Run the command with a `--name` flag to add an `increment` contract to our project. From inside our `soroban-hello-world` directory, run:
 
 ```sh
-stellar contract init ./ --with-example increment
+stellar contract init . --name increment
```

Comment:
I think we should add something about how the user should copy and paste the Increment contract code into their newly created contract file. But I am happy to make that change when I open a new PR with some additional tweaks.

Reply:
Added.